### ISSUE TITLE
Supporting dark-light pairs

### ISSUE THREAD

**Issue by FranklinYu** [tinted-theming/tinted-iterm2#9]
This is a feature request.

The beta version of iTerm2 supports dual-mode themes, where one theme consists of light and dark variants. iTerm2 comes with 3 of such themes. It would be nice if this repository can generate such themes.

See also mbadolato/iTerm2-Color-Schemes#372.

**Comment by JamyGolden** [comment]
What does this allow one to do? Change theme based on time of day?

**Comment by FranklinYu** [comment]
> What does this allow one to do? Change theme based on time of day?

Yes, iTerm2 will automate the change of theme. It can be based on time of day, or based on OS preferences (from macOS).
